### PR TITLE
Aviation fix formatting and finish conversion to JSON (from Avro)

### LINKED ISSUE #1051: Suggestions to improve aviation recipe

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#1051 

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/aviation-fix/

### New tutorial checklist

### CODE REVIEW

**Comment by rspurgeon** [comment]
@ybyzek Looks good. One other thing this recipe could do is change the casing of the field names in the statements.

For example:
```
CREATE TABLE customers (
  ID INT PRIMARY KEY,
  NAME VARCHAR,
  ADDRESS VARCHAR,
  EMAIL VARCHAR,
  PHONE VARCHAR,
  LOYALTY_STATUS VARCHAR
``` 
would be more consistent with the rest of the recipes as
```
CREATE TABLE customers (
  id INT PRIMARY KEY,
  name VARCHAR,
  address VARCHAR,
  email VARCHAR,
  phone VARCHAR,
  loyalty_status VARCHAR
```

This is also preferred based on the ksqlDB SME reviews highlighted in this comment: https://github.com/confluentinc/ksqldb-recipes/pull/160#discussion_r795899031

File: _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql [hunk 1]
```diff
@@ -1,90 +1,95 @@
...
 );
 
 CREATE TABLE customer_bookings AS 
   SELECT C.*, B.ID, B.FLIGHT_ID
-  FROM   bookings B
-          INNER JOIN customers C
-              ON B.CUSTOMER_ID = C.ID;
+  FROM bookings B
+  INNER JOIN customers C
+  ON B.CUSTOMER_ID = C.ID;
```
Comment (rspurgeon) [inline _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql:43]: Something to consider and just my personal opinion:  it doesn't really help the reader to abbreviate table and stream names like this. `bookings` and `customers` aren't long complicated names and `B` and `C` aren't very helpful abbreviated names in their own right.

Comment (ybyzek) [reply _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql]: Let's leave as-is for now.

**Comment by ybyzek** [comment]
> @ybyzek Looks good. One other thing this recipe could do is change the casing of the field names in the statements.
> 
> For example:
> 
> ```
> CREATE TABLE customers (
>   ID INT PRIMARY KEY,
>   NAME VARCHAR,
>   ADDRESS VARCHAR,
>   EMAIL VARCHAR,
>   PHONE VARCHAR,
>   LOYALTY_STATUS VARCHAR
> ```
> 
> would be more consistent with the rest of the recipes as
> 
> ```
> CREATE TABLE customers (
>   id INT PRIMARY KEY,
>   name VARCHAR,
>   address VARCHAR,
>   email VARCHAR,
>   phone VARCHAR,
>   loyalty_status VARCHAR
> ```
> 
> This is also preferred based on the ksqlDB SME reviews highlighted in this comment: [confluentinc/ksqldb-recipes#160 (comment)](https://github.com/confluentinc/ksqldb-recipes/pull/160#discussion_r795899031)

Agree and fixed in latest commit

**Review by rspurgeon — APPROVED** [review]

File: _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql [hunk 2]
```diff
@@ -21,22 +21,22 @@ CREATE TABLE flights (
...
-  KAFKA_TOPIC='bookings',
-  FORMAT='JSON',
-  PARTITIONS=6
+  KAFKA_TOPIC = 'bookings',
+  FORMAT = 'JSON',
+  PARTITIONS = 6
 );
 
-CREATE TABLE customer_bookings AS 
+CREATE TABLE customer_bookings WITH (KAFKA_TOPIC = 'customer_bookings') AS
```
Comment (rspurgeon) [inline _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql:39]: does this need a partition and/or format configuration?

File: _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql [hunk 3]
```diff
@@ -66,8 +84,23 @@ CREATE STREAM cf_rekey WITH (KAFKA_TOPIC = 'cf_rekey') AS
...
+  customer_phone VARCHAR,
+  customer_loyalty_status VARCHAR,
+  flight_origin VARCHAR,
+  flight_destination VARCHAR,
+  flight_code VARCHAR,
+  flight_scheduled_dep TIMESTAMP,
+  flight_scheduled_arr TIMESTAMP
+) WITH (
+  KAFKA_TOPIC = 'cf_rekey',
+  FORMAT = 'JSON'
```
Comment (rspurgeon) [inline _includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql:102]: partition configuration?

**Pull request merged**